cinema/serializers.py

```diff
@@ -30,10 +30,31 @@ class Meta:
         fields = ("id", "name", "rows", "seats_in_row", "capacity")
 
 
+class MovieImageSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Movie
+        fields = ("id", "image")
```

Comment:
Code Efficiency issue: It seems you're trying to serialize an image as a model field. Consider using `serializers.ImageField` instead of `serializers.ModelSerializer` for the image field. This would be more efficient and appropriate for handling image data. Here's an example: 
python
image = serializers.ImageField(max_length=None, use_url=True)
```